Action.c

```diff
@@ -401,15 +401,19 @@ static Htop_Reaction actionPrevScreen(State* st) {
 }
 
 Htop_Reaction Action_setScreenTab(State* st, int x) {
+   assert(x >= 0);
+   if (x < 0)
+      return 0;
+   unsigned int pos = x;
    Settings* settings = st->host->settings;
-   int s = 2;
+   unsigned int s = 2;
```

Comment:
I think for this algorithm, it's better for `s` to be in `size_t`, and use `(size_t)x` (upcast) for comparisons below. I think I would submit my propsed change in a separate PR.